From 05ed1d5953388de067ca441c7bdd4704b35351a3 Mon Sep 17 00:00:00 2001
From: Andrew Ammerlaan <andrewammerlaan@gentoo.org>
Date: Fri, 26 Apr 2024 21:41:26 +0200
Subject: [PATCH] snapd-qt/meson.build: Install Notice header

Otherwise compilation of Discover with snap back-end fails with:

In file included from /usr/include/snapd-qt-2/Snapd/Client:1,
from /tmp/portage/kde-plasma/discover-6.0.4-r1/work/discover-6.0.4/libdiscover/backends/SnapBackend/libsnapclient/SnapAuthHelper.cpp:14:
/usr/include/snapd-qt-2/Snapd/client.h:24:10: fatal error: Snapd/Notice: No such file or directory
24 | #include <Snapd/Notice>
|          ^~~~~~~~~~~~~~

Signed-off-by: Andrew Ammerlaan <andrewammerlaan@gentoo.org>
---
 snapd-qt/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/snapd-qt/meson.build b/snapd-qt/meson.build
index a767c6d..db2f5bf 100644
--- a/snapd-qt/meson.build
+++ b/snapd-qt/meson.build
@@ -131,6 +131,7 @@ source_alias_h = [
   'Snapd/MarkdownNode',
   'Snapd/MarkdownParser',
   'Snapd/Media',
+  'Snapd/Notice',
   'Snapd/Plug',
   'Snapd/PlugRef',
   'Snapd/Price',
-- 
2.46.0

